docs/ReleaseHistory.md

```diff
@@ -24,7 +24,13 @@
 - NEW: Sort properties by name in GeneratedRegexPatterns/*.json.
 - BRK: Remove derived `RegexPattern` class properties `ChecksumSeeds`, `EncodeForUrl`, and `KeyLength` as these are not relevant to the literal authoring of equivalent regex patterns in other languages.
 - BRK: Rename `LegacyCommonAnnotatedSecurityKey` to `CommonAnnotatedSecurityKey` and mark this inlined `Microsoft.Security.Utilities` class as internal.
+- BRK: Remove methods on `CachedDotNetRegex` that are not supported by the common `IRegexEngine` interface.
+- BRK: Direct use of `CachedDotNetRegex` no longer forces `RegexOptions.Compiled` and `RegexOptions.NonBacktracking`. It is up to the caller to pass them in if overriding the default argument.
+- BRK: Rename `RegexDefaults.DefaultOptionsCaseSensitive` to `RegexDefaults.DefaultOptions`.
+- BRK: Remove `RegexPattern.DefaultRegexOptions`. Use `RegexDefaults.DefaultOptions` instead.
+- BRK: `RegexOptions.CultureInvariant` is now used by default.
 - NEW: Provide deterministic ordering of properties in GeneratedRegexPatterns/*.json via `DataMember` attributes.
+- NEW: It is now possible to turn off default regex options by passing `disabledRegexOptions` to the `RegexPattern` constructor.
```

Comment:
>regex [](http://example.com/codeflow?start=46&length=5)

Remove this now yes?

Reply:
yes, thanks

Reply:
we should also add a `BRK` on the conversion to a nullable type for the relevant ctor arg.